docs/adapters/Withdraw.md

```diff
@@ -0,0 +1,70 @@
+## Adapter description and scope
```

Comment:
@jdville03 Updated the Withdraw.md with the proper information.

Reply:
Looks good, but I think the Functions section looks like it just has the placeholders from the Template.

Reply:
Ah I see that it just has some leftovers from the Template. I'll clean that up in the PR I'm submitting to your branch.